sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -1843,6 +1843,26 @@ std::vector<device_image_plain> ProgramManager::getSYCLDeviceImages(
   return DeviceImages;
 }
 
+void setSpecializationConstants(
```

Comment:
```suggestion
static void setSpecializationConstants(
```

Or do we have custom processing of what's internal/external?

Reply:
You're right. This would likely have caused problems on Windows. Added!